game_app/games/tic_tac_toe/tic_tac_toe_component.py

```diff
@@ -165,19 +165,32 @@ def __init__(self, app, board_size, marks_required, player_mark, opponent_mark,
         players = self.server.players
 
         # TicTacToeClientActor server joining
-        p0 = players[self._player_mark]
-        self.tell(self.client_actor_address, JoinServerMsg(p0))
+        human_player = players[self._player_mark]
+        self.tell(self.client_actor_address, JoinServerMsg(human_player))
 
         # TODO use self._board_size, self.marks_required, self._difficulty to load proper agent
         # Opponent joining
-        p1 = players[self._opponent_mark]
-        second_player_file_path = os.path.join(ABS_PROJECT_ROOT_PATH,
-                                               "reinforcement_learning",
-                                               "agents",
-                                               f"second_player_q_ep_0.ai")
-
-        c1 = AgentClient(BaseAgent.load(second_player_file_path))
-        self.server.join(c1, p1)
+        # File choosing
```

Comment:
ten komentarz chyba moze wyleciec, zwlaszcza jak przerzucisz to do funkcji nazwanej resolve_agents_file_name